patches/tModLoader/Terraria.UI/ItemSlot.cs.patch

```diff
@@ -53,6 +53,75 @@
  				{
  					if (Main.mouseRightRelease)
  					{
+ 						player.OpenBossBag(inv[slot].type);
+-						inv[slot].stack--;
```

Comment:
In regards to the earlier comment, do not turn these nested because it will increase the patch difference.